syz-manager/html.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/google/syzkaller/cover"
 	. "github.com/google/syzkaller/pkg/log"
 	"github.com/google/syzkaller/prog"
+	. "github.com/google/syzkaller/report"
```

Comment:
Don't add more dot imports. That's bad taste.

Yes, we use it for log. But let's stop here. Log was dot-imported to avoid large diffs. Let's better remove dot import of log.